lib/migration.js

```diff
@@ -517,29 +378,9 @@ function mixinMigration(PostgreSQL) {
     return colDefault ? (' DEFAULT ' + colDefault) : '';
   };
 
-  /*!
-   * Find the column type for a given model property
-   *
-   * @param {String} model The model name
-   * @param {String} property The property name
-   * @returns {String} The column type
-   */
-  PostgreSQL.prototype.columnDataType = function(model, property) {
```

Comment:
Moved `columnDataType` to base connector https://github.com/strongloop/loopback-connector/pull/78/commits/752359febbc39abb5ecc4c38cb15ac18012c1112 
We still have `buildColumnType` in postgresql